### ISSUE TITLE
Backslashes in custom commands not escaped in JSON

### ISSUE THREAD

**Issue by Zguy** [deepnight/ldtk#955]
I have a project with a custom command that contains backslashes.
When I save the project, the command is saved unescaped in the JSON file
```
"customCommands": [{ "command": "out\build\x64-Debug\bin\App.exe", "when": "Manual" }],
```
This is not valid JSON.

When I try to open the project, I get the following error
```
Cannot read properties of null (reading 'externalLevels') (TypeError)
```
Which is an incredibly misleading error. `externalLevels` is nowhere close to `customCommands` physically within the JSON file.

If I manually escape the backslashes in the custom command with a text editor I can open the project again, but every time I save it's saved unescaped again.

**Comment by deepnight** [comment]
Thanks for reporting this bug!

**Comment by deepnight** [comment]
Fixed in 1.4
